csbuild/_build/project_plan.py

```diff
@@ -337,10 +337,8 @@ def ExecutePlan(self, toolchainName, architectureName, targetName):
 				"all"
 			)
 
-			for depend in flattenedDepends:
-				# pylint: disable=protected-access
-				dependObj = allPlans[depend]
-				# type: ProjectPlan
+			for depend in flattenedDepends: # pylint: disable=protected-access
```

Comment:
This is a weird change to me. The disable was supposed to affect this entire for block since there's multiple protected-access instances here. Moving it to the end of the line makes it affect only this line, which isn't accessing anything protected. Why did you move it?

Reply:
The old disabled error still does the same thing.  Appending it to the line still has the effect of applying to the sub-block of that line, so nothing has changed in that regard.